bundles/org.openhab.binding.argoclima/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,501 @@
+				<label>Argo Remote Server Address</label>
+				<description><![CDATA[The OEM server's hostname, used for communications.<br/>
+				<i>This is the same host as for the Web Application (address provided in device's user manual). Example: uisetup.ddns.net</i>
+				]]>
+				</description>
+				<default>31.14.128.210</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="oemServerPort" type="integer" min="0" max="65535" required="false"
+				groupName="oemServerConnection">
+				<label>Argo Remote Server Port</label>
+				<description><![CDATA[The OEM server's port. <i>Default 80.</i>]]>
+				</description>
+				<default>80</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="ac-controls-group">
+		<label>A/C Controls</label>
+		<category>HVAC</category>
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="mode" typeId="mode-basic-channel"/>
+			<channel id="set-temperature" typeId="set-temperature-channel"/>
+			<channel id="current-temperature" typeId="system.indoor-temperature"/>
+			<channel id="fan-speed" typeId="fan-speed-channel"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="modes-group">
+		<label>Operation Modes</label>
+		<channels>
+			<channel id="eco-mode" typeId="eco-mode-modifier-channel">
+				<label>Eco Mode</label>
+			</channel>
+			<channel id="turbo-mode" typeId="turbo-mode-modifier-channel">
+				<label>Turbo Mode</label>
+			</channel>
+			<channel id="night-mode" typeId="night-mode-modifier-channel">
+				<label>Night Mode</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="timers-group">
+		<label>Timers</label>
+		<category>Time</category>
+		<channels>
+			<channel id="active-timer" typeId="active-timer-channel"/>
+			<channel id="delay-timer" typeId="delay-timer-channel"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="unsupported-group">
+		<label>Unsupported Modes</label>
+		<description><![CDATA[These channels are available on the original Argo remote (and app), but the Ulisse DCI device does not support it.]]></description>
+		<channels>
+			<channel id="mode-ex" typeId="mode-extended-channel"/>
+			<channel id="swing-mode" typeId="swing-mode-channel"/>
+			<channel id="filter-mode" typeId="filter-mode-modifier-channel">
+				<label>Filter Mode</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="settings-group">
+		<label>Settings</label>
+		<channels>
+			<channel id="ifeel-enabled" typeId="ifeel-channel"/>
+			<channel id="device-lights" typeId="device-lights-channel"/>
+			<channel id="temperature-display-unit" typeId="temperature-display-unit-channel"/>
+			<channel id="eco-power-limit" typeId="eco-power-limit-channel"/>
+		</channels>
+	</channel-group-type>
+
+
+	<!--
+		*** CHANNELS ***
+	-->
+	<channel-type id="mode-basic-channel">
```

Comment:
These channel type id suffixes are unneeded (all).
```suggestion
	<channel-type id="mode-basic">
```

Reply:
ACK unneded (& changed).

----

For the record, I'm not a fan of using the same name for 2 types (and letting semantics decide which is which), ex like this
```xml
<channel id="set-temperature" typeId="set-temperature"/>
<!-- note: sure could invent 2 distinct names, w/o resorting to suffixes but in some cases it is hard -->
````
...because it makes searching for what I want in less powerful editors (or grep) a little more involved. 
But I searched through other addon's codebase and see that I'm quite alone in that statement, so... rename it is 😉